nb/updatecenters/extras/nbjavac.api/release/modules/ext/nb-javac-15-api.jar.external

```diff
@@ -1,6 +1,5 @@
-CRC:2157920864
-SIZE:252431
-URL:https://netbeans.osuosl.org/binaries/267AFC87A08301145B10D86DA674724B884F010C-nb-javac-15-api.jar
-URL:https://hg.netbeans.org/binaries/267AFC87A08301145B10D86DA674724B884F010C-nb-javac-15-api.jar
-MessageDigest: SHA-256 0a5e9b6afbdb86deb3286f37e1dee230c91f39438e55dc77910c9a48f38a2ffb
-MessageDigest: SHA-512 5b16a0c6466dc14cf2d428d4a84398f9de2d1782acf445c8da5ca874f807ac550db8b551ec41495963a29c2f400ca24ee8a70b1d589d0cdf2f05af5cb0f4ff56
\ No newline at end of file
+CRC:2454529625
+SIZE:3700020
+URL:https://github.com/oracle/nb-javac/releases/download/release130/0788681E011A58CE17E5583718F4919A-nb-javac-15.0.0.1-impl.jar
```

Comment:
Let's use the `m2:` "Maven URLs" here too. Please find an example of such approach in our [Java FX libraries integration](https://github.com/apache/netbeans/blob/master/extra/libs.javafx.linux/external/javafx-media-13-linux.jar.external#L2).